src/functionality/configUtils.ts

```diff
@@ -63,17 +80,22 @@ export function getSharedTheme(
  */
 
 export function updateViewsCalciteMode(mode: "light" | "dark"): void {
-  const prevCalciteMode = mode === "light" ? "calcite-mode-dark" : "calcite-mode-light";
-  const calciteMode = mode === "light" ? "calcite-mode-light" : "calcite-mode-dark";
+  const prevCalciteMode =
+    mode === "light" ? "calcite-mode-dark" : "calcite-mode-light";
```

Comment:
values used more than once can be stored in a const i.e. `mode === 'light'`, `calcite-mode-light`, `calcite-mode-dark`, etc.